environment.yml

```diff
@@ -5,57 +5,66 @@ channels:
   - plotly
   - defaults
 dependencies:
+  # mdanalysis dependencies
   - cython
-  - distributed==2021.1.0
-  - dask==2021.1.0
-  - python-graphviz
-  - pytest
-  - decorator
-  - docutils
-  - graphviz
-  - ipywidgets<8.0.0
-  - ipython
-  - ipygany
-  - jedi
-  - jinja2
-  - jupyter_contrib_nbextensions
   - matplotlib
-  - nbclient
-  - nbconvert
-  - nbformat
-  - nbsphinx
   - networkx
-  - nglview
-  - notebook
   - numpy
+  - openmm>=7.0.0
   - packaging
   - parmed
-  - parso
-  - pip
-  - plotly
-  - pybtex
-  - python=3.9
-  - pyvista
-  - scikit-image
   - scikit-learn>=1.1.2
+  - threadpoolctl
+  # building documentation
+  - docutils
+  - pandoc
+  - pybtex
   - sphinx
   - sphinxcontrib-bibtex
-  - threadpoolctl
+  # used by custom_parallel_analysis.ipynb
+  - graphviz
   - tabulate
-  - widgetsnbextension
-  - ipycytoscape>=1.3.0
+  - python-graphviz
+  # used in multiple notebooks
```

Comment:
I would label this section generally `extensions` instead. `nbconvert` especially I would assume is not called by any notebook explicitly but to download notebooks, or in nbsphinx.

Reply:
Thank you these changes were easy to apply